src/backend/oracle-data-api/src/main/java/ca/bc/gov/open/jag/tco/oracledataapi/model/TicketImageDataJustinDocument.java

```diff
@@ -0,0 +1,62 @@
+package ca.bc.gov.open.jag.tco.oracledataapi.model;
+
+import java.sql.Blob;
+
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+//mark class as an Entity
+/**
+ * @author 237563
+ *
+ *         Represents JJ Document retrieved from justin
+ *
+ */
+//defining class name as Table name
```

Comment:
you could remove these comments as they only apply to H2 which we are no longer using.